diff --git a/include/spdlog/common.h b/include/spdlog/common.h
index aca483c2..5e39c744 100644
--- a/include/spdlog/common.h
+++ b/include/spdlog/common.h
@@ -52,7 +52,7 @@
 #if !defined(SPDLOG_USE_STD_FORMAT) && \
     FMT_VERSION >= 80000  // backward compatibility with fmt versions older than 8
     #define SPDLOG_FMT_RUNTIME(format_string) fmt::runtime(format_string)
-    #define SPDLOG_FMT_STRING(format_string) FMT_STRING(format_string)
+    #define SPDLOG_FMT_STRING(format_string) fmt::runtime(format_string)
     #if defined(SPDLOG_WCHAR_FILENAMES) || defined(SPDLOG_WCHAR_TO_UTF8_SUPPORT)
         #include <spdlog/fmt/xchar.h>
     #endif
